dashboard/src/main/resources/webapp/static/lib/angular-topology-editor/canvas.js

```diff
@@ -101,7 +101,8 @@ Graph.Node.popovercontent = function(i) {
 };
 
 var Canvas = (function() {
-
+    var LOG = Log.getLogger("Canvas").setLevel(Log.INFO);
```

Comment:
@adriannieto, all js libraries export variables as global to access their code:

* jquery exports $ and jQuery
* d3 exports d3
* our canvas, editor, etc, export Canvas, Editor...

loglevel does the same, and even gives the option to choose the global name you need to avoid conflicts.